app/src/main/java/co/nimblehq/compose/ecommerce/ui/screen/home/Home.kt

```diff
@@ -1,36 +1,189 @@
+                style = AppTextStyle.title
+            )
+
+            BadgeBox(
+                modifier = Modifier.constrainAs(bbProfile) {
+                    end.linkTo(parent.end)
+                    bottom.linkTo(parent.bottom)
+                },
+                badgeContent = {
+                    Text(
+                        "3",
+                        style = AppTextStyle.badge
+                    )
+                },
+                backgroundColor = Red
+            ) {
+                Image(
+                    painter = painterResource(id = R.drawable.ic_profile),
+                    contentDescription = "Profile",
+                    modifier = Modifier.size(38.dp)
+                )
+            }
+        }
+
+        ConstraintLayout(
+            modifier = Modifier
+                .fillMaxWidth()
+                .height(386.dp)
+                .clip(shape)
+                .background(Blue)
+        ) {
+            val (tvSuggestForYou, tvCollection, tvPrice, btShop, sDivider, ivShirt, tvPoly) = createRefs()
+
+            // FIXME Correct the image to match design
+            Image(
+                modifier = Modifier
+                    .constrainAs(ivShirt) {
+                        start.linkTo(parent.start)
+                        top.linkTo(parent.top)
+                    }
+                    .size(419.dp, 378.dp)
+                    .padding(top = 92.dp, start = 32.dp),
+                painter = painterResource(id = R.drawable.ic_shirt),
+                contentDescription = "collection"
+            )
+
+            Text(
+                modifier = Modifier
+                    .constrainAs(tvSuggestForYou) {
+                        top.linkTo(parent.top)
+                        start.linkTo(parent.start)
+                    }
+                    .padding(top = 16.dp, start = 16.dp),
+                text = stringResource(R.string.home_suggested_for_you),
+                style = AppTextStyle.tagline
+            )
+
+            Text(
+                modifier = Modifier
+                    .constrainAs(tvCollection) {
+                        top.linkTo(tvSuggestForYou.bottom)
+                        start.linkTo(tvSuggestForYou.start)
+                    }
+                    .padding(top = 4.dp, start = 16.dp, end = 16.dp),
+                text = stringResource(R.string.home_collection),
+                style = AppTextStyle.collection
+            )
+
+            // Work around to display the divider with padding
+            Column(modifier = Modifier
+                .padding(horizontal = 16.dp)
+                .constrainAs(sDivider) {
+                    bottom.linkTo(btShop.top)
+                    start.linkTo(parent.start)
+                    end.linkTo(parent.end)
+                }
+            ) {
+                Spacer(
+                    modifier = Modifier
+                        .background(Color.White.copy(alpha = 0.33f))
```

Comment:
Let's put it in the `Color.kt` -> `WhiteAlpha33`

Reply:
@Wadeewee It's good to extract the color to the Color file but it's not being reused in other places, so that's why I decided to use `copy` with `alpha` for efficiency.